.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,16 @@
 ### Description
 [Describe what this change achieves]
- 
-### Issues Resolved
-[List any issues this PR will resolve]
- 
+
+### Related Issues
+Resolves #[Issue number to be closed when this PR is merged]
+<!-- List any other related issues here -->
+
 ### Check List
 - [ ] New functionality includes testing.
-  - [ ] All tests pass, including unit test, integration test and doctest
 - [ ] New functionality has been documented.
-  - [ ] New functionality has javadoc added
-  - [ ] New functionality has user manual doc added
```

Comment:
Could we keep these two sub-checkboxes? Or merge into one line such as
```
[ ] New functionality has been documented(javadoc, [user manual doc](https://github.com/opensearch-project/sql/blob/main/DEVELOPER_GUIDE.rst#reference-manual).
```

They are the actual items to explain what is `New functionality has been documented` meaning. As a new contributor, it's really useful to me.

Reply:
I re-added those as they were before. The links will vary per branch, it's a different can of worms, leaving it to ya'll for a later improvement.